packages/reports/addon/templates/components/grouped-list.hbs

```diff
@@ -1,15 +1,32 @@
-{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-{{#each-in groupedItems as | group items |}}
-  <details class="grouped-list__group" open={{shouldOpenAllGroups}}>
-    <summary class="fa grouped-list__group-header">
-      <span class="grouped-list__group-header-content">{{group}} ({{items.length}})</span>
-    </summary>
-    <ul>
-      {{#each items as |item|}}
-        <li class="grouped-list__item">
-          {{yield item}}
-        </li>
-      {{/each}}
-    </ul>
-  </details>
-{{/each-in}}
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="{{this.groupedListClass}}" ...attributes>
+  <VerticalCollection
+    @items={{this.flatItems}}
+    @tagName="ul"
+    @estimateHeight={{or @estimateHeight 17}}
+    @staticHeight={{if (eq @staticHeight false) false true}}
```

Comment:
```suggestion
    @staticHeight={{not (eq @staticHeight false)}}
```
Would this work?

Reply:
I don't like reading this 😢but it should work

Reply:
ember-arg-types 👀